### PR TITLE
Add two quotes for the status messages

### PR DESCRIPTION
**Pull request by jeremy0x**
Reference - #12

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -58,4 +58,6 @@
    "Not good with words? Offer a hug. Hugs are the ultimate source of comfort.",
    "Be kind. Be lovable. Be huggable.",
    "Hugging is the silent way of saying \"You matter to me\".",
+   "Be a love pharmacist; dispense hugs like medicine — they are!","Be a love pharmacist; dispense hugs like medicine — they are!",
```
Comment (achasveachas) [inline statuses.py:61]: Looks like you accidentally pasted this twice?

**Review by achasveachas — CHANGES_REQUESTED** [review]
Looks great 🤗 

Address the typo and then I will merge.

File: statuses.py (hunk 1 above)
Comment (jeremy0x) [reply statuses.py]: Yeah, it's been fixed now.

**Comment by jeremy0x** [comment]
> Address the typo and then I will merge.

typo corrected ✔️

**Review by achasveachas — APPROVED** [review]
Looks great, thanks! 🤗 

Let me know if you have a twitter handle so I can give you a shout-out.

**Comment by jeremy0x** [comment]
Thank you @achasveachas!

> Let me know if you have a twitter handle so I can give you a shout-out.

My twitter handle is [@jeremy0x_](https://twitter.com/jeremy0x_).

**Pull request merged**